PinpointKit/PinpointKit/Sources/InterfaceCustomization.swift

```diff
@@ -86,6 +90,7 @@ public struct InterfaceCustomization {
         public init(tintColor: UIColor? = UIColor.pinpointOrangeColor(),
                     annotationFillColor: UIColor? = nil,
                     annotationStrokeColor: UIColor = .whiteColor(),
+                    annotationTextAttributes: [String: AnyObject]? = nil,
```

Comment:
Instead of having a separate property for the font (`editorTextAnnotationFont `), could you please remove that property, and use `annotationTextAttributes` to specify the font?

We’re currently using `updateInterfaceCustomization()` in `EditImageViewController` to set the font for the text view. In this function, we could pull the `NSFontAttributeName` value out of the `annotationTextAttributes`, and apply it the same way.